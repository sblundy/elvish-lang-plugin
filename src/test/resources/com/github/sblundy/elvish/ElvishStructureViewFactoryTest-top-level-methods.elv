fn test-method {
  echo ok
}