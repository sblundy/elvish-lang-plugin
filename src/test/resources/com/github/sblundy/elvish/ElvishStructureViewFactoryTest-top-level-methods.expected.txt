-ElvishStructureViewFactoryTest-top-level-methods.elv
 test-method